packages/styled/flow-tests/base.js

```diff
@@ -1,23 +1,17 @@
 /* eslint-disable no-unused-vars */
 // @flow
-import * as React from 'react'
+import React from 'react'
 import createStyled from '../src/base'
 import type { CreateStyledComponent, StyledComponent } from '../src/utils'
 
-export const valid: CreateStyledComponent = createStyled('div')
+const valid: StyledComponent<empty> = createStyled('div')({
```

Comment:
Don't ask me how it used to work before, but `CreateStyled` return type is not `CreateStyledComponent`, it's `(...args: Interpolations) => StyledComponent<Props>` (defined inline).

To make the test simpler, I call `createStyled` to generate `(...args: Interpolations) => StyledComponent<Props>`, and then I call it again, to generate a Styled Component. Then I check the returned type is `StyledComponent`.

The result is pretty much the same.